iphone/Classes/TiUITabGroup.m

```diff
@@ -413,6 +413,34 @@ - (void)hideTabBar:(BOOL)hidden animated:(BOOL)animated
       }];
 }
 
+#if IS_SDK_IOS_26
+- (void)setBottomAccessoryView_:(id)bottomAccessoryViewProxy
+{
+  if (bottomAccessoryViewProxy == [NSNull null]) {
+    [[self tabController] setBottomAccessory:nil];
+    return;
+  }
```

Comment:
If this had a bottom accessory view before you also need to forget the proxy and release the view here.